tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -438,3 +441,77 @@ async def mockup(*args):
     with running_backend.offline():
         with pytest.raises(FSBackendOfflineError):
             await alice_workspace.get_reencryption_need()
+
+
+@pytest.mark.trio
+async def test_upload_blocks(alice_user_fs, alice2_user_fs, running_backend, monkeypatch):
+    wid = await create_shared_workspace("w", alice_user_fs, alice2_user_fs)
```

Comment:
no need for this heavy helper function here given we don't have to share the workspace with another user (given alice and alice2 are two devices of the same user)
So we can do instead:
```python
wid = await alice_user_fs.workspace_create("w")
await alice_user_fs.sync()
await alice2_user_fs.sync()
```